### ISSUE TITLE
Add a function to get file paths

### ISSUE THREAD

**Issue by water111** [water111/jak-project#7]
It would be nice to have an easy way to get a path to a file inside of `jak-project` that works in any of the C++ tools and takes care of any windows/linux differences.   I think this should be written in `common/util` and there should be a `util` library so all the programs can use it.

One possible version could work like this:
```
std::string path = get_file_path("goal_src", "kernel", "gkernel.gc");
```
and it would return an absolute file path to `goal_src/kernel/gkernel.gc` that's formatted correctly on Windows and Linux. It could use the `NEXT_DIR` environment variable that will be set in scripts that launch tools, or maybe there is a better way to do this. 

Once this is in place, we should remove the `FAKE_ISO_PATH` variable that's set/read in various scripts, and use this new function in `fake_iso.cpp`.
